src/backend/catalog/heap.c

```diff
@@ -1654,6 +1654,8 @@ heap_create_with_catalog(const char *relname,
 	 * NAMEDATALEN and gets truncated. Then the name may same with other child
 	 * table's.
 	 *
+	 * Auxiliary relations in pg_ext_aux don't need array type.
```

Comment:
It's weird. Should we create type under extaux_namespace or not?

Reply:
Added them.